hq/app/AppComponents.scala

```diff
@@ -107,6 +109,7 @@ class AppComponents(context: Context)
   private val efsClients = AWS.efsClients(configuration, availableRegions)
   val securityCredentialsProvider =
     new AWSCredentialsProviderChain(new ProfileCredentialsProvider("security"), DefaultAWSCredentialsProviderChain.getInstance())
+  private val securityS3Client = AmazonS3ClientBuilder.standard().withRegion("eu-west-1").withCredentials(securityCredentialsProvider).build()
```

Comment:
There's already a mechanism above for getting hold of s3 clients, would it be better to re-use that? Creating instances of AWS clients comes with a bit of overhead - IIRC each has its own threadpool for example. If so, better to re-use the existing cached client created in the `AWS.s3Clients` call, just above.